secure-ingress-cognito/envs/dev/values.yaml

```diff
@@ -0,0 +1,17 @@
+labels:
+  env: dev
+spec:
+  destination:
+    server: https://kubernetes.default.svc
+  source:
+    repoURL: https://github.com/aws-samples/eks-blueprints-workloads
+    targetRevision: main
+
+  # Thoses default values can be surcharged by Infrastructure as Code
+  env: dev
+  clusterName: eks-blueprint-cluster # the name of EKS cluster
+  blueprint: 'terraform' # cdk | terraform
```

Comment:
what is the significance of stating that it is a terraform blueprint (or cdk)?

Reply:
was thinking of adding this to terraform repo in future but I have removed it now.